.eslintrc.js

```diff
@@ -7,8 +7,8 @@ module.exports = {
             [
                 '',
                 {
-                    pattern: '^ \\* Copyright \\(c\\) \\d{4}, salesforce.com, inc\\.$',
-                    template: ' * Copyright (c) 2021, salesforce.com, inc.'
+                    pattern: '^ \\* Copyright \\(c\\) \\d{4}, (salesforce.com, inc|Salesforce, Inc)\\.$',
+                    template: ' * Copyright (c) 2022, salesforce.com, inc.'
```

Comment:
Dumb question, should it be a range? starting from 2021 - 2022?

Reply:
Nope, just the current year is fine.